vector/src/main/res/layout/activity_home.xml

```diff
@@ -30,74 +34,130 @@
+            app:layout_constraintTop_toBottomOf="@+id/home_recents_sync_in_progress" />
 
         <!-- The main content view -->
-        <RelativeLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:layout_above="@+id/bottom_navigation"
-            android:layout_below="@+id/listView_pending_callview"
-            tools:context="org.matrix.vector.activity.VectorHomeActivity">
-
-            <FrameLayout
-                android:id="@+id/fragment_container"
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:background="?attr/riot_primary_background_color" />
+        <FrameLayout
+            android:id="@+id/fragment_container"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:layout_constraintTop_toBottomOf="@id/listView_pending_callview"
+            app:layout_constraintBottom_toTopOf="@+id/bottom_navigation"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            android:background="?attr/riot_primary_background_color" />
+
+        <FrameLayout
+            android:id="@+id/listView_spinner_views"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:layout_constraintTop_toBottomOf="@id/listView_pending_callview"
+            app:layout_constraintBottom_toTopOf="@+id/bottom_navigation"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            android:background="?attr/waiting_background_color"
+            android:visibility="gone">
+
+            <ProgressBar
+                android:id="@+id/listView_spinner"
+                android:layout_width="40dp"
+                android:layout_height="40dp"
+                android:layout_gravity="center"
+                android:visibility="visible" />
+        </FrameLayout>
 
-            <android.support.design.widget.FloatingActionButton
-                android:id="@+id/floating_action_button"
+        <!-- Elevate button above the touch guard (above lollipop). -->
+        <com.getbase.floatingactionbutton.FloatingActionsMenu
+            android:id="@+id/floating_action_menu"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintBottom_toTopOf="@id/bottom_navigation"
+            android:elevation="12dp"
+            app:fab_addButtonColorNormal="@color/vector_fuchsia_color"
+            app:fab_expandDirection="up"
+            app:fab_labelStyle="@style/Floating_Actions_Menu"
+            app:backgroundTint="@color/vector_fuchsia_color"
+            app:borderWidth="0dp"
+            app:elevation="4dp">
```

Comment:
So is the desired elevation 4dp or 12dp? I think the 2 values should be identical.